src/lib/db/feature-toggle-store.ts

```diff
@@ -482,6 +474,21 @@ export default class FeatureToggleStore implements IFeatureToggleStore {
 
         return result?.potentially_stale ?? false;
     }
+
+    private columnsWithMetrics = () => {
```

Comment:
Do we need metrics on all of these store methods? This is quite expensive, making a new database call for each feature.

Reply:
I tried to add the column only for UI - admin - where do you think it is not needed?